### ISSUE TITLE
Inline anchors in section titles are not resolved properly

### ISSUE THREAD

**Issue by slonopotamus** [asciidoctor/asciidoctor-intellij-plugin#469]
book.adoc:

```adoc
= Title

== Section 1

Link to <<foo>>.

== [[foo]]Section 2

Text.
```

Expected: document renders without errors, clicking on a link navigates to section 2

Actual:

![image](https://user-images.githubusercontent.com/92637/80306829-640fef00-87ce-11ea-84ba-2b432e89683f.png)

AsciiDoc Plugin Version: 0.30.59.1

IDE Name and Version: RubyMine 2020.1
Build #RM-201.6668.156, built on April 14, 2020
Runtime version: 11.0.6+8-b765.25 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0

**Comment by ahus1** [comment]
Thanks for reporting this. The plugin doesn't support inline IDs in headers and other places very well as of today. 

In the case above you could switch to the following syntax: 

```
[[foo]]
== Section 2
```

This will automatically replace `<<foo>>` with the title of the section. 

A future version of the plugin will support this.

**Comment by mojavelinux** [comment]
The underlying problem here is the placement of the anchor. Putting an anchor at the start of the title is a non-standard location. For it to be recognized by the core parser, it must be at the end of the section title.

```
== Section 2 [[foo]]
```

The anchor at the beginning is treated like any other inline anchor, meaning it's just a jump point. It's not associated with the section title in the model.

**Comment by ahus1** [comment]
Status update: Pre-release 0.30.68 is available that parses anchors in normal text, but not in section headers yet (b83e808106e9a61992498bdf7169739410c8d963)

**Comment by ahus1** [comment]
Handling anchors in section titles will be available in the next upcoming release 0.31.28

**Comment by ahus1** [comment]
This is now available as part of pre-release 0.31.28.

The pre-release of the plugin is available from [GitHub releases](https://github.com/asciidoctor/asciidoctor-intellij-plugin/releases) and the [IntelliJ AsciiDoc EAP repository](https://intellij-asciidoc-plugin.ahus1.de/docs/contributors-guide/user/add-eap-repository-to-ide.html).
